.mailmap

```diff
@@ -185,6 +185,8 @@
 #
 # See also: 'git shortlog --help' and 'git check-mailmap --help'.
 #
+
+Aadya Chinubhai <aadyachinubhai@gmail.com> Aadya Chinubhai <77720426+aadya940@users.noreply.github.com>
```

Comment:
After adding your name here you should run the `bin/mailmap_check.py` script which will reorder the lines in the .mailmap file.

Reply:
The file is required to be in alphabetical ordering to avoid merge conflicts.